lisa/runner.py

```diff
@@ -167,7 +179,10 @@ def _generate_runners(self, runbook: schema.Runbook) -> Iterator[BaseRunner]:
             # keep filters to current runner's only.
             runbook.testcase = parse_testcase_filters(raw_filters)
             runner = factory.create_by_type_name(
-                type_name=runner_name, runbook=runbook, index=len(self._runners)
+                type_name=runner_name,
+                runbook=runbook,
+                index=len(self._runners),
+                case_variables=case_variables,
```

Comment:
Might as well inline that call here

Reply:
If so, that would be called multiple times in the loop.

Reply:
Oops, disregard. /me grabs more coffee